radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/CreateTokenToParticleGroupsMapper.java

```diff
@@ -66,7 +66,7 @@
 				TokenUnitConversions.unitsToSubunits(tokenCreation.getInitialSupply()),
 				TokenUnitConversions.unitsToSubunits(tokenCreation.getGranularity()),
 				tokenCreation.getAddress(),
-				System.currentTimeMillis(),
+				System.nanoTime(),
```

Comment:
Why nano time?

Reply:
Found that using currentTimeMillis could sometimes return back the same if called in succession close to each other.

Reply:
Also, currentTimeMillis() is often quantised as well.  On some OS's only updated in units of 18ms or so.